haystack/pipelines/standard_pipelines.py

```diff
@@ -91,7 +91,7 @@ def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None, overwri
         Here's a sample configuration:
 
             ```yaml
-            |   version: '0.8'
+            |   version: '1.0.0'
```

Comment:
I changed it only because I think 0.8 does not exist, while 1.0.0 is the first one that does. No real reason though :smile: I find untidy that we have many different versions in the examples so I always go with 1.0.0. Totally subjective.